docs/examples/getting-started.md

```diff
@@ -0,0 +1,125 @@
+# Tendermint
+
+## Overview
+
+Tendermint is ...
+
+## Install
+
+Note: Quick Start on a fresh machine can be done with [this script](https://gist.githubusercontent.com/zramsay/4bca8907a97ad825c30cfdc6f72ba97b/raw/b94bf36e34181a2960f9fe0d3eff260d03b6064e/install_tendermint.sh); tested on Ubuntu 16.04. It is used in the cluster deployment below.
```

Comment:
let's add a command to run

```
curl -L https://gist.githubusercontent.com/zramsay/4bca8907a97ad825c30cfdc6f72ba97b/raw/b94bf36e34181a2960f9fe0d3eff260d03b6064e/install_tendermint.sh | bash
```
 or
```
curl -L https://git.io/vNTiV | bash
```

Reply:
Note: this link will change to refer to the file currently being check-in with this PR, however, that file should point to master branch, so that can wait until release